otter/test/models/test_cass_models.py

```diff
@@ -3745,30 +3766,43 @@ def test_get_servers_diff_last_update(self):
         """
         dt_earlier = datetime(2010, 10, 15, 10, 0, 0)
         self._test_get_servers(
-            [{"server_blob": '{"a": "b"}', "last_update": dt_earlier},
-             {"server_blob": '{"d": "e"}', "last_update": dt_earlier},
-             {"server_blob": '{"c": "f"}', "last_update": self.dt}],
+            False,
+            [{"server_blob": '{"a": "b"}', "last_update": dt_earlier,
+              "server_as_active": False},
+             {"server_blob": '{"d": "e"}', "last_update": dt_earlier,
+              "server_as_active": False},
+             {"server_blob": '{"c": "f"}', "last_update": self.dt,
+              "server_as_active": False}],
             ([{"a": "b"}, {"d": "e"}], dt_earlier))
+        # Test with only_as_active as True
+        self._test_get_servers(
+            True,
+            [{"server_blob": '{"a": "b"}', "last_update": dt_earlier,
+              "server_as_active": False},
+             {"server_blob": '{"d": "e"}', "last_update": dt_earlier,
+              "server_as_active": True},
+             {"server_blob": '{"c": "f"}', "last_update": self.dt,
+              "server_as_active": True}],
+            ([{"d": "e"}], dt_earlier))
```

Comment:
Yes. you are right! The test is incorrectly written which unfortunately doesnt show that the code is incorrect. The code will still work since it will return list ordered differently, i.e. the list returned will have later date first which is what the test should do. I'll change it. Thanks for catching this!